velox/connectors/hive/HiveConnector.h

```diff
@@ -92,46 +96,130 @@ class HiveTableHandle : public ConnectorTableHandle {
   const core::TypedExprPtr remainingFilter_;
 };
 
+/// Location related properties of the Hive table to be written
+class LocationHandle {
+ public:
+  enum class TableType {
+    kNew, // Write to a new table to be created.
+    kExisting, // Write to an existing table.
+    kTemporary, // Write to a temporary table.
```

Comment:
```
nit: s/kTemporary/kTemp/
```
What's the use of temporary table?

Reply:
Presto OSS discourage abbrev. As discussed, temporary is a table type of Hive.